_security-plugin/authentication-backends/authc-index.md

```diff
@@ -1,23 +1,28 @@
 ---
 layout: default
-title: Authentication flow
-parent: Configuration
-nav_order: 1
+title: Authentication backends
+nav_order: 45
+has_children: true
+has_toc: false
+redirect_from:
+  - /security-plugin/authentication-backends/
 ---
 
-# Authentication flow
+# Authentication backends
 
-Understanding the authentication flow is a great way to get started with configuring the security plugin.
+Authentication backend configurations determine the method or methods you use for authenticating users and the way users pass their credentials and sign in to OpenSearch. Having an understanding of the basic authentication flow before getting started can help with the configuration process for whichever backend you choose. Consider the high-level sequence of events in the description that follows, and then refer to the detailed steps for configuring the authentication type you choose to use with OpenSearch.
 
-1. To identify a user who wants to access the cluster, the security plugin needs the user's credentials.
+## Authentication flow
+
+1. To identify a user who wants to access the cluster, the Security plugin needs the user's credentials.
 
    These credentials differ depending on how you've configured the plugin. For example, if you use basic authentication, the credentials are a user name and password. If you use a JSON web token, the credentials are stored within the token itself. If you use TLS certificates, the credentials are the distinguished name (DN) of the certificate.
 
-2. The security plugin authenticates the user's credentials against a backend: the internal user database, Lightweight Directory Access Protocol (LDAP), Active Directory, Kerberos, or JSON web tokens.
+2. The Security plugin authenticates the user's credentials against a backend: the internal user database (basic authentictation), Lightweight Directory Access Protocol (LDAP)/Active Directory, JSON web tokens, SAML, or another authentication protocol.
```

Comment:
Thinking about how to be precise and clear when communication about backend vs. way of communicating auth info (user + password combo or token) to OpenSearch.

There are 2 high-level types of IdP, Identity Providers, backends that Opensearch security plugin interfaces with. 1. Internal User database or 2. an external provider (a backend compliant with LDAP, ODIC or SAML). For client cert, jwt and proxy auth, these could be described as backend-less providers (not completely accurate, but I'll explain) in that OpenSearch does not verify the info passed in the request with a list of users and instead extracts username and backend roles from the request passed to opensearch. 

1. For client cert auth, the principal of the cert is used to extract the username and backend roles. I believe username is the `CN` field and `backend_roles` are the concatenated `OU` fields. As long as the cert is issued by a CA in the `SECURITY_SSL_HTTP_PEMTRUSTEDCAS_FILEPATH` setting the cert will be trusted.
2. For JWT, a token is passed in an HTTP Authorization request header which must contain claims that match the `subject_key` and `roles_key`. The JWT must also be able to be verified with the `signing_key` configured in `config.yml`
3. For proxy auth authentication is performed at the IdP and then forwarded to opensearch. The forwarded request contains the username and backend roles in the HTTP request and OpenSearch trusts the request if its coming from the correct configured source.

The internal database is used with Basic Auth to get username and password info from an HTTP request. So in this case **Basic Auth** is the mechanism for transmitting auth info (username + password) in a request and the **Authentication Backend** would be the internal user database. 

How does this sound?

```
The Security plugin authenticates a request against a configured backend.  Authentication providers include: Basic Auth using the internal user database, Lightweight Directory Access Protocol (LDAP)/Active Directory, JSON web tokens, SAML, or another authentication protocol.
```

This change you are presenting in this PR looks good to me, but stuff to keep in mind.

Reply:
I see the distinction. And those are good points to keep in mind. I tweaked the suggestion a little. Does it still make sense, or have I taken it off course?

`The Security plugin authenticates a server request against a backend configured for an authentication provider.  Some examples of authentication providers used with OpenSearch include Basic Auth (which uses the internal user database), Lightweight Directory Access Protocol (LDAP)/Active Directory, JSON web tokens, SAML, or another authentication protocol.`

Reply:
@cwillum The tweaked suggestion sounds good to me, but instead of `server request` can it simply be `request`?

Judging by item 1, the word Credentials is used generically to mean what's needed to confer access so its accurate to say that credentials must be included on every request. 

For most of the backends, Username + Password will be used for the very first request so its accurate to say the credentials are included in the request. In many cases, subsequent requests will use an access token (JWT) to prevent the need of passing username + password on every request.

In item 1:

> the credentials are stored within the token itself

When I first read this it sounded like password info was embedded in the token, but that's not the case. The username (`subject_key`) and roles (`roles_key`) are embedded in the token.

Reply:
RE: "... but instead of server request can it simply be request?"
Definitely. I'm probably unnecessarily splitting hairs with some of these proposed changes to wording. But it's helpful to refine along the way. Thanks for your patience.